### ISSUE TITLE
Simplify links to folders for JS and no-JS situations.

### ISSUE THREAD

**Issue by dcavins** [boonebgorges/buddypress-docs#514]
Our current template includes two versions of the link to a folder: one vanilla HTML for non-JS use, and another pretty vanilla for JS use.

@boonebgorges I'm not sure of the historical reasons for the hide-if-js/hide-if-no-js parallel structures, but I'm wondering if we could use an approach like this, where JS kicks in if it's available, but the basic structure works fine without it: https://github.com/dcavins/buddypress-docs/commit/1be0a84e02ab114029537f0b89ccf76c4bdc98bf

**Comment by boonebgorges** [comment]
I don't remember the historical reasons. You could `git blame` to see if 
I (or you) left a meaningful commit message at some point.

I'm not certain that screen readers will have a specific problem with 
the "duplicate" links. I believe screen readers generally parse CSS in 
meaningful ways. This should be tested before any sweeping changes are made.

That said, if you have a way of making things work smoothly for non-JS 
without repetition, go ahead and do it.

**Comment by dcavins** [comment]
Fangs (a screen reader emulator) only parses the visible link, which is great.

This section of code was added by me here: https://github.com/boonebgorges/buddypress-docs/commit/fc47ffd7fab2b6a957e0caf3f992dc82c3accd33. I followed an existing pattern used in the docs templates.

If neither of us can think of any reason to use two links, let's use the simpler version, which works fine with JS and no-JS (I've tested both).
